LICENSE

```diff
@@ -19,3 +19,6 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+Contibuting Authors:
+Advay Mengle
```

Comment:
Should we have a separate CONTRIBUTORS file?

Reply:
Depends on the CLA, let's put it here and move later if it gets long?  We are doing MIT not APACHE so there's no NOTICE file, so this may be the only file copied into the final build.

Reply:
That's fine then